.docs/README.md

```diff
@@ -0,0 +1,223 @@
+# Console Bridge
+
+# Content
+- [Usage - how to use it](#usage)
+    - [Register one by one](#usage)
+    - [Register ConsoleBridge](#usage)
+- [Extensions - list of all extensions](#extension)
+    - [Cache](#CacheConsole)
+    - [Caching](#CachingConsole)
+    - [DI](#DIConsole)
+    - [Latte](#LatteConsole)
+    - [Router](#RouterConsole)
+    - [Security](#SecurityConsole)
+    - [Utils](#UtilsConsole)
+    
+## Usage
+
+**Register commands one by one:**
+```yaml
+extensions:
+    cacheConsole: Contributte\Console\Extra\DI\CacheConsoleExtension
+    cachingConsole: Contributte\Console\Extra\DI\CachingConsoleExtension
+    diConsole: Contributte\Console\Extra\DI\DIConsoleExtension
+    latteConsole: Contributte\Console\Extra\DI\LatteConsoleExtension
+    routerConsole: Contributte\Console\Extra\DI\RouterConsoleExtension
+    securityConsole: Contributte\Console\Extra\DI\SecurityConsoleExtension
+    utilsConsole: Contributte\Console\Extra\DI\UtilsConsoleExtension
+```
+You can install just one or multiple, but if you want all, we recommend **ConsoleBridge**.
+
+
+**Register ConsoleBridge:**
+```yaml
+exntesions:
+    consoleBridge: Contributte\Console\Extra\DI\consoleBridgeExtension
+```
+
+The ConsoleBridge registers all the extensions automatically, you just have to register the `ConsoleBridgeExtension`
+
+To use these commands we recommend to use an **[entrypoint](https://github.com/contributte/console/blob/master/.docs/README.md#entrypoint)** or you can use your `index.php`.
+
+With [entrypoint](https://github.com/contributte/console/blob/master/.docs/README.md#entrypoint):
+```bash
+./bin/console <command>
+```
+without [entrypoint](https://github.com/contributte/console/blob/master/.docs/README.md#entrypoint):
+```bash
+php public/index.php <command>
```

Comment:
What does it mean without entrypoint? :-)

Reply:
it sounds weird now. I ment it without `bin/console` over the `index.php`, but from the definition it is entrypoint either, going to fix it with different words